### DISCUSSION TITLE
Get-PnPNavigationNode always returns IsVisible property as true

### DISCUSSION THREAD

**Discussion by GregBirbrayer** [pnp/powershell#4484] [General]
### Discussed in https://github.com/pnp/powershell/discussions/4482

<div type='discussions-op-text'>

<sup>Originally posted by **GregBirbrayer** October 27, 2024</sup>
Get-PnPNavigationNode returns all the navigation nodes, however if the navigation link is hidden (sp onPrem) it still returns the IsHidden prop as $true. 
This is causing the confusion when migrating the quick launch menu to sp online adding up extra nav nodes.</div>

**Comment by jackpoz** [comment]
Which version of PnP PowerShell are you using ? Because the current version supports only SharePoint Online.

**Reply by GregBirbrayer** [reply]
What's the command to check the PnP Powershell version? Thanks.

**Reply by GregBirbrayer** [reply]
I got it - it is 2.12.0

Which version should I import for having IsHidden property functionable? I am in the process of migrating quick launch menus from on prem to sp online. Thank you much.

**Reply by jackpoz** [reply]
The readme at https://github.com/pnp/powershell?tab=readme-ov-file#pnp-powershell mentions that this repo supports only SharePoint Online, with a link to the old https://github.com/pnp/pnp-powershell that supported SP2013/2016/2019.
https://github.com/pnp/PnP-PowerShell/releases/tag/3.26.2010.0 has a "How to install" section with how to install different versions from PowerShell Gallery.
